keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -199,13 +200,35 @@ def __init__(
 
         super().__init__(**kwargs)
 
+        if vocabulary_size != None and isinstance(vocabulary_size, int) == False:
```

Comment:
I think we can probably do without this error, we don't have errors for `lowercase` being a bool for example.